superset/examples/configs/charts/Featured Charts/Graph_124.yaml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+slice_name: Graph
+description: null
+certified_by: null
+certification_details: null
+viz_type: graph_chart
+params:
+  datasource: 22__table
+  viz_type: graph_chart
+  source: deal_size
+  target: product_line
+  metric: count
+  adhoc_filters:
+    - clause: WHERE
+      subject: order_date
+      operator: TEMPORAL_RANGE
+      comparator: No filter
+      expressionType: SIMPLE
+  row_limit: 10000
+  color_scheme: supersetColors
```

Comment:
For some reason the color palette seems different for this chart in your screenshot.

Reply:
This is the chart's default color palette. My screenshot is using the dashboard's color palette which overrides every chart in it.